contrib/platform/test/com/sun/jna/platform/win32/Kernel32Test.java

```diff
@@ -539,21 +539,6 @@ public void testCreateProcess() {
         assertTrue(processInformation.dwProcessId.longValue() > 0);
     }
 
-    public void testGetEnvironmentVariable() {
```

Comment:
Oh I see you moved them into `EnvironmentVarsTest.java`, that makes it very hard to find, please stick to the convention of Kernel32.java -> Kernel32Test.java and Kernel32Util.java -> Kernel32UtilTest.java (and I would be ok with Kernel32Util_EnvironmentVarsTest.java).